src/controllers/buyProduct.controller.js

```diff
@@ -0,0 +1,57 @@
+import Cart from '../models/cart';
+import Product from '../models/Product';
+
+export class buyProduct {
+  static async getPdts(req, res, next) {
```

Comment:
Already this functionality is done to fetch all products.

Reply:
I'll remove this then.